source/Playnite/Database/GameDatabase.cs

```diff
@@ -1310,6 +1313,37 @@ bool updateCompletionStatus(Game game, CompletionStatusSettings settings)
             }
         }
 
+        public List<FilterPreset> GetSortedFilterPresets(bool getClone)
+        {
+            var filterPresetsSettings = (FilterPresets as FilterPresetsCollection).GetSettings();
+            var sortingOrder = filterPresetsSettings.SortingOrder;
+            var filterPresets = getClone ? FilterPresets.GetClone() : FilterPresets;
+            return filterPresets.OrderBy(x =>
+            {
+                var index = sortingOrder.IndexOf(x.Id);
+                if (index == -1)
+                {
+                    // The max value is returned to make the item
+                    // show last in order
+                    return sortingOrder.Count;
+                }
+                else
+                {
+                    return index;
+                }
+            }).ToList();

```

Comment:
That getClone thing is imo completely unnecessary since you are returning completely new List here anyways.

Reply:
Parameter removed in https://github.com/JosefNemec/Playnite/pull/3258/commits/4d9ec25d9bf201e2aece128f6c87cc257eda7b60